FBC/FunctionalEntities/BusinessCenters.rdf

```diff
@@ -66,14 +66,15 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>business center code</rdfs:label>
-		<skos:definition>a code used to denote a municipality or business center</skos:definition>
+		<skos:definition>code used to denote a metropolitan area where business is conducted</skos:definition>
```

Comment:
Not sure this captures the notion of "business center" as used in FIBO where there is more of a emphasis on financial business activity; 
And the Code is in the FpML Scheme (should we not have an individual of class Scheme?)